contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -129,7 +129,6 @@ contract BasicAMBErc677ToErc677 is
         external
         onlyIfUpgradeabilityOwner
     {
-        require(!fixedAssets(messageId));
         require(valueToUnlock <= maxPerTx());
```

Comment:
It looks like the check that `valueToUnlock` is within the limit is required only if `unlockOnForeign` is true.

Reply:
Yes, makes sense